src/components/InstallationPage/extensionStatus.js

```diff
@@ -7,25 +8,37 @@ export const extensionStatuses = {
 }
 
 export const useExtensionStatus = () => {
-  const [installed, setInstalled] = useState(extensionStatuses.checking)
+  const [installed, setInstalled] = useState(extensionStatuses.notInstalled)
 
   useEffect(() => {
     const checkExtensionInstallation = () => {
-      const root = document.getElementById('cozy-app')
-      const isExtensionInstalled = root.dataset.cozyExtensionInstalled
+      window.postMessage(
+        {
+          message: {
+            app: 'cozy-passwords',
+            url: window.location.toString()
+          }
+        },
+        '*'
```

Comment:
With that `*`, every app / page can listen to it. It it wanted? 

It seems that on Chrome you have a dedicated API to do so : https://developer.chrome.com/extensions/messaging#external-webpage

Reply:
I added a comment explaining why we are doing this. We are looking for a better solution meanwhile.